.github/workflows/pr_validate.yaml

```diff
@@ -0,0 +1,138 @@
+name: Validate pull request
+on:
+  pull_request:
+    types: [ opened, reopened, edited ]
+jobs:
+  # BEWARE: GitHub uses Microsoft-style line endings (CRLF)
+  # instead of Unix-style (LF) interally for things like PR text.
+  #
+  # You MUST check for `\r\n` as the line separator instead of `\n`
+  # in this workflow.
+  lint-pr-body:
+    name: Lint pull request body
+    runs-on: ubuntu-20.04
+    concurrency: 
+      group: ${{ github.head_ref }}
+      cancel-in-progress: true
+    outputs:
+      sandbox: ${{ steps.extract-staging.outputs.sandbox }}
+      should_e2e_run: ${{ steps.check-e2e-trigger.outcome }}
+    steps:
+      - id: install
+        name: Install requirements
+        run: |-
+          sudo apt-get update -y
+          sudo apt-get install -y pcregrep
+
+      - id: check-jira-url
+        name: Check JIRA URL
+        run: |-
+          REGEX="#### URL to JIRA issue\s?\r\n(https:\/\/biomage\.atlassian\.net\/browse\/BIOMAGE-[0-9]+|N\/A)"
+
+          echo "REGEX to test against:"
+          echo $REGEX
+          echo "Now looking at regex result:"
+
+          pcregrep -o1 -M "$REGEX" <<\EOF
+          ${{ github.event.pull_request.body }}
+          EOF
```

Comment:
why do you need this?

Reply:
We cannot couple JIRA with out build system, because that prevents open source contributors to merge pull requests unless they are added to jira

Reply:
As we discussed, this is not necessary, you can leave it unfilled by adding N/A. This way the system knows that you deliberately ignored it instead of forgetting to fill it in.

Reply:
That will not be a problem, you can replace the JIRA URL with N/A and it will pass. It's just to make sure you are deliberately not including one.

Reply:
I did remove all references to JIRA though (outside of the actual validator obviously) so when we change to GItHub we can just replace the regex.